test/tests.py

```diff
@@ -915,8 +915,8 @@ def test(self):
         super().test()
         dir = Path(self.scratchdir) / 'dumped'
         dir.mkdir()
-        with tarfile.open(Path(self.scratchdir) / f'T_{self.shortname}.tar.xz',
-                          'r:xz') as tf:
+        with tarfile.open(Path(self.scratchdir) / f'T_{self.shortname}.tar.gz',
+                          'r:gz') as tf:
```

Comment:
The corresponding `tarfile.open` that created the file is in `py/dml/dmlc.py`.

The xz usage is for an odd use case and it's not important that it works for all pythons, so from a usability perspective it's OK if this test breaks. However, if switching to a different compression scheme makes something easier then I'm OK with that. I'd prefer bz2 over gz, though.

Reply:
I'm setting up our CI to run all our tests with some host Pythons, and then run into this. An alternative is to build all the Pythons used to link with liblzma, or to allow XFAILing based on which Python was used.

Reply:
The choice of format is unimportant; switching from xz to bz2/gz is fine if that makes anything easier in CI.

Reply:
It appears that the Pythons in our lab that I was planning to use all have bz2 support, so I could use that instead if you prefer that.